Improve the public code of conduct at website (#8)
pythoncoruna/pythoncoruna.github.io: content/en/code-of-conduct/index.md

that do not align with this code of conduct, and will communicate the reasons for its moderation
decisions when appropriate.

## CoC violation protocol

Cases of abusive, harassing, or otherwise unacceptable behavior can be reported to the community
administrators who are responsible for enforcement.

> If you feel that some of the above described behaviours have happened or you feel that any offense has been made, you can take the next course of action:

Reviewer: ```suggestion
If you believe that any of the described behaviors have occurred or if you feel offended, you can take the following course of action:
```
